lib/SGN/Controller/Search/Cross.pm

```diff
@@ -76,12 +76,12 @@ sub download_cross_entries : Path('/search/download_cross_entries') Args(0) {
 
     my $cross_properties_string = $c->config->{cross_properties};
     my @cross_properties = split ',', $cross_properties_string;
-    my $file_format = "xls";
+    my $file_format = "xlsx";
 
     my $time = DateTime->now();
     my $timestamp = $time->ymd();
     my $dir = $c->tempfiles_subdir('download');
-    my $temp_file_name = "cross_entries". "XXXX";
```

Comment:
it is probably better to keep the XXXX for random file generation

Reply:
Great catch - I will check it. Thank you.